extensions/commands/sbom/cmd_cyclonedx.py

```diff
@@ -151,12 +154,16 @@ def me_as_tool() -> Tool:
                                                          remotes, args.update)
     # endregion COPY
 
-    components = {node: create_component(node) for node in deps_graph.nodes}
+    def filter_context(node): return not args.no_build_requires or node.context != CONTEXT_BUILD
```

Comment:
```suggestion
    def filter_context(n): return not args.no_build_requires or n.context != CONTEXT_BUILD
```
It is generally better to avoid using arguments named as existing variables in scope